code/ship/ship.cpp

```diff
@@ -3665,6 +3674,28 @@ static void parse_ship_values(ship_info* sip, const bool is_template, const bool
 			sip->closeup_zoom = 0.5f;
 		}
 	}
+
+	if(optional_string("$Closeup_pos_targetbox:"))
+	{
+		stuff_vec3d(&sip->closeup_pos_targetbox);
+	} 
+	else 
```

Comment:
You need to use `else if (first_time)` here.  Otherwise, if you have a ships.tbl and a modular table, and that modular table doesn't have a `$Closeup_pos_targetbox:` entry, it will always be reset to the default.  Take a look at `$Closeup_pos:` above.